.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "markdown_metayaml"]
```

Comment:
I see, this is in fact a "chain" of two plugins.

As far as I've tested, YAML in Markdown files works well just by using pelican with settings:
```python
MD_EXTENSIONS = [
    ...
    'md_meta_yaml.meta_yaml',
    ...
]
```
(`md_meta_yaml` being the referenced Markdown plugin.)

So if _Markdown plugin_ works, is there a need for _Pelican_ plugin as well? :neutral_face: